doc/user/installation.rst

```diff
@@ -289,6 +289,91 @@ FRR will run with any kernel configuration but some recommendations do exist.
    (:ref:`rip`) or *ospfd* (:ref:`ospfv2`) because these protocols use
    multicast.
 
+Linux sysctl settings and kernel modules
+````````````````````````````````````````
+
+There are several kernel parameters that impact overall operation of FRR when
+using Linux as a router. Generally these parameters should be set in a
+sysctl related configuration file, e.g., :file:`/etc/sysctl.conf` on
+Ubuntu based systems and a new file
+:file:`/etc/sysctl.d/90-routing-sysctl.conf` on Centos based systems.
+Additional kernel modules are also needed to support MPLS forwarding.
+
+:makevar:`IPv4 and IPv6 forwarding`
```

Comment:
I'm guessing a copy-paste error on the `makevar`

Reply:
@qlyoung which is?? Please be specific with either how you'd like it changed or what specific error you are seeing.